### PR TITLE
ui fixes in header container

### PR DESCRIPTION
**Pull request by Rajamouli-hcl**
ui changes in headercontainer and font updated in filtercomponent

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/css.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+less/header.less
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/css.txt]: @Rajamouli-hcl : file name should be container.less

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/header.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+document.querySelector(".search i.cmp-search__icon").addEventListener("click", headerSearch);
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/header.js]: @Rajamouli-hcl : file name should be container.js

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml [hunk 3]
```diff
@@ -18,6 +18,20 @@
                     cq:policy="hclecomm/components/structure/search/policy_1589453975454"
                     jcr:primaryType="nt:unstructured"
                     sling:resourceType="wcm/core/components/policies/mapping"/>
+                <hclecomm jcr:primaryType="nt:unstructured">
+                    <components jcr:primaryType="nt:unstructured">
+                        <content jcr:primaryType="nt:unstructured">
+                            <button
+                                cq:policy="hclecomm/components/content/button/policy_1588076880690"
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml]: @Rajamouli-hcl : why is there a new policy for button component ?

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
@Rajamouli-hcl : Pl check comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/css.txt (hunk 1 above)
Comment (Rajamouli-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/css.txt]: updated to container.less

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/header.js (hunk 2 above)
Comment (Rajamouli-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/header.js]: updated to container.js

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml (hunk 3 above)
Comment (Rajamouli-hcl) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml]: removed button policy

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Pull request merged**